strongbox-aql/src/test/java/org/carlspring/strongbox/aql/AqlParserTest.java

```diff
@@ -61,14 +61,14 @@ public void testLayoutSpecificKeywords()
 
         logger.debug(String.format("Query [%s] parse tree:\n[%s]", query, aqlParser));
 
-        assertTrue(aqlParser.hasErrors());
-        assertNotNull(errorMap);
-        assertEquals(2, errorMap.size());
+        assertThat(aqlParser.hasErrors()).isTrue();
+        assertThat(errorMap).isNotNull();
+        assertThat(errorMap).hasSize(2);
 
         List<Pair<Integer, Integer>> errorPositionList = new ArrayList<>(errorMap.keySet());
 
-        assertEquals(Pair.with(1, 55), errorPositionList.get(0));
-        assertEquals(Pair.with(1, 115), errorPositionList.get(1));
+        assertThat(Pair.with(1, 55).equals(errorPositionList.get(0))).isTrue();
```

Comment:
`assertThat(Pair.with(1, 55)).isEqualTo(errorPositionList.get(0));` is not applicable here since this is ambiguous call (`asssertThat(Iterable<?>) `and `assertThat(Pair<Integer, Integer>)`).

Refactored other `equals()` usages.